SFS_general/modal_weighting.m

```diff
@@ -114,7 +114,12 @@
 
 % Inverse DTFT
 if nargout>1
-    Win = ifft([win,zeros(1,order)],ndtft,'symmetric');
+    if mod(ndtft,2)
+        win = [win, win(end:-1:2)];
+    else
+        win = [win, win(end-1:-1:2)];
+    end
+    Win = ifft(win);
```

Comment:
The new code does not mimic the old code.  try e.g. ``[win, Win] = modal_weighting(10, 1024, conf)`` before and after the changes and check ``Win``